### PR TITLE
Add variable for focused input background

### PR DESCRIPTION
**Pull request by aronstrandberg**
Add a variable to make it possible to configure the input's background color when focused. 

I was trying to theme the focused state in a project, but found it bit cumbersome, hence the desire for these changes. I'd be happy to make any requested changes from your side.

### CODE REVIEW

**Comment by JedWatson** [comment]
Thanks @aronstrandberg, happy to accept this addition.

Would you please add it to the `scss` files as well though? we need to keep them in sync with the `less`, unfortunately this is a manual process.

File: less/select.less [hunk 1]
```diff
@@ -15,6 +15,7 @@
 // control options
 @select-input-bg:                   #fff;
 @select-input-bg-disabled:          #f9f9f9;
+@select-input-bg-focus:             #fff;
```
Comment (JedWatson) [inline less/select.less]: Please make this default to `@select-input-bg` in case that has been customised, otherwise this would be a breaking change.

**Comment by aronstrandberg** [comment]
@JedWatson Pushed some commits to adress these things. I couldn't find the equivalent of `.Select-focus-state` in the scss files, so I'm not 100%, but I think they should be similar now – is there a way to test the examples with the scss files to make sure?

**Comment by aronstrandberg** [comment]
@JedWatson I just rebased this on master to adjust for upstream changes. Do you think this is ready to merge?

**Comment by aronstrandberg** [comment]
@JedWatson?

**Comment by JedWatson** [comment]
Looks great @aronstrandberg, thanks!

**Comment by abdennour** [comment]
Do not release feature with big bug : 

```
Failed to compile.

Module build failed: 
		background: $select-input-bg-focus;
             ^
      Undefined variable: "$select-input-bg-focus".
      in /Users/abdennour/workspace/idax/node_modules/react-select/scss/control.scss (line 70, column 15)

```

**Pull request merged**